packages/tooling/__tests__/lib/parameters-to-json-schema.test.js

```diff
@@ -5,26 +5,26 @@ const fixtures = require('../__fixtures__/lib/json-schema');
 const polymorphismScenarios = ['oneOf', 'allOf', 'anyOf'];
 
 test('it should return with null if there are no parameters', async () => {
-  expect(parametersToJsonSchema({ parameters: [] })).toBeNull();
-  expect(parametersToJsonSchema({})).toBeNull();
+  expect(await parametersToJsonSchema({ parameters: [] })).toBeNull();
+  expect(await parametersToJsonSchema({})).toBeNull();
 });
 
 describe('parameters', () => {
   describe('type sorting', () => {
     const schema = {
       parameters: [
-        { in: 'path', name: 'path parameter' },
-        { in: 'query', name: 'query parameter' },
-        { in: 'header', name: 'header parameter' },
-        { in: 'cookie', name: 'cookie parameter' },
+        { in: 'path', name: 'path parameter', schema: { type: 'string' } },
+        { in: 'query', name: 'query parameter', schema: { type: 'string' } },
+        { in: 'header', name: 'header parameter', schema: { type: 'string' } },
+        { in: 'cookie', name: 'cookie parameter', schema: { type: 'string' } },
```

Comment:
Added `schema` properties to these because it was invalid OAS and wouldn't pass `@openapi-contrib/openapi-schema-to-json-schema`.